CMakeLists.txt

```
@@ -73,10 +74,15 @@ message(STATUS "HEXL_EXPERIMENTAL:             ${HEXL_EXPERIMENTAL}")
 message(STATUS "HEXL_SHARED_LIB:               ${HEXL_SHARED_LIB}")
 message(STATUS "HEXL_TESTING:                  ${HEXL_TESTING}")
 message(STATUS "HEXL_TREAT_WARNING_AS_ERROR:   ${HEXL_TREAT_WARNING_AS_ERROR}")
+message(STATUS "HEXL_FPGA_COMPAT:              ${HEXL_FPGA_COMPAT}")
 
 hexl_check_compiler_version()
 hexl_add_compiler_definition()
 
+if (HEXL_FPGA_COMPAT)
+  add_definitions(-DHEXL_FPGA_COMPAT)
```

This function has been superseded by `add_compile_definitions`
```suggestion
  add_compile_definitions(HEXL_FPGA_COMPAT)
```